Runtime/Plugins/Raycast/UIWidgetsPanelRaycastFilter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Unity.UIWidgets.engine;
+using UnityEngine;
+
+namespace Unity.UIWidgets.plugins.raycast {
+    
+    [DisallowMultipleComponent]
+    [RequireComponent(typeof(UIWidgetsPanel))]
+    [DefaultExecutionOrder(1)]
+    public class UIWidgetsPanelRaycastFilter : MonoBehaviour, ICanvasRaycastFilter {
+        public bool reversed;
+        
+        UIWidgetsPanel panel;
+        int windowHashCode;
+
+        void OnEnable() {
```

Comment:
Maybe we should also un-register the window's raycast Rect List when it is destroyed or disabled ?